config/services/dataplane_v1beta1_openstackdataplaneservice_configurenetwork.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  labels:
+    app.kubernetes.io/name: openstackdataplaneservice
+    app.kubernetes.io/instance: openstackdataplaneservice-sample
+    app.kubernetes.io/part-of: dataplane-operator
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: dataplane-operator
+  name: configure-network
+spec:
+  label: dataplane-deployment-configure-network
+  role:
+
+    name: "Deploy EDPM Network"
+    hosts: "all"
+    strategy: "linear"
+    tasks:
+      - name: "Install edpm_bootstrap"
+			  import_role:
```

Comment:
I can't create this CR in my enviornment:

[fultonj@osp-storage-01 composable]$ oc create -f ~/dataplane-operator/config/services/dataplane_v1beta1_openstackdataplaneservice_configurenetwork.yaml
error: error parsing /home/fultonj/dataplane-operator/config/services/dataplane_v1beta1_openstackdataplaneservice_configurenetwork.yaml: error converting YAML to JSON: yaml: line 19: did not find expected key
[fultonj@osp-storage-01 composable]$ 

I see (using whitespace mode in my editor) that there are four tabs and then two spaces before the first `import_role` on line 20. I'll see if I can fix locally and share.

I passed it to https://www.yamllint.com and it said:

```
Tabs are not allowed as indentation at line 20, column 1
Tabs are not allowed as indentation at line 23, column 1
Tabs are not allowed as indentation at line 24, column 1
A block sequence may not be used as an implicit map key at line 26, column 1
Implicit keys need to be on a single line at line 26, column 7
Tabs are not allowed as indentation at line 30, column 1
Implicit map keys need to be followed by map values at line 26, column 7
```

Reply:
should be fixed now.